### PR TITLE
translate 2015-05-05-swift-documentation.md

### PR DESCRIPTION
**Pull request by aprilnov**

### CODE REVIEW

**Comment by aprilnov** [comment]
@croath @rickytan thx~

**Comment by croath** [comment]
这么快，顶不住人民群众的巨大压力了...

**Comment by aprilnov** [comment]
刘老板说的周五交货哒... ^ ^

**Comment by croath** [comment]
（囧（心虚（我自己还没开始搞（ :lollipop:

**Comment by natecook1000** [comment]
Quick turnaround! :clap: Would it make sense to translate some of the comments, themselves, in this case? Whatever would communicate the documentation best practices would be great.

**Comment by aprilnov** [comment]
@croath @rickytan 😈

**Comment by croath** [comment]
How about @natecook1000 's suggestion, translate some comments in the article?

**Comment by aprilnov** [comment]
I think that's already pretty straight and simple... And, we don't recommend devs to write their comments in Chinese, right? (Just image how terrifying that would be..😱

**Comment by croath** [comment]
@aprilnov But... it may not be that straight to everyone, so there we are translating it.

And I'm not that agree that it's not commended to write comments in localized language. To let everyone feel free to read and understand is the point, isn't it? BTW, you could even name your variables with CJK characters or Emoji in Swift...

**Comment by aprilnov** [comment]
@croath OK for me, will translate later then..

I'm now working for a Korean company.. they always writes comments in Korean, which already made me craaaazy~~  That's why I don't recommend that...you never know the nationality of whom will read your code...

**Comment by croath** [comment]
@aprilnov Yep I do agree it's better to write comments in English. Maybe we should write a comment of "comments" in the article to explain that we translate them to Chinese on purpose but English is recommended.

**Comment by aprilnov** [comment]
@croath Yep..I add a comment at the bottom of it, please review. Thanks.

**Comment by croath** [comment]
看笑了，翻译的真不错，LGTM

@rickytan

File: 2015-05-05-swift-documentation.md [hunk 1]
```diff
@@ -0,0 +1,327 @@
+---
+title: Swift Documentation
+author: Mattt Thompson & Nate Cook
+authors:
+    - Mattt Thompson
+    - Nate Cook
```
Comment (croath) [inline 2015-05-05-swift-documentation.md]: @natecook1000 It seems a bug here that the `author` showing on nishipster.cn are all `April Peng`(the translator)

**Pull request merged**